public/docs/dart/latest/guide/architecture.jade

```diff
@@ -1,451 +1,51 @@
-
-  * The `(click)` [event binding](user-input.html#click) calls the component's `selectHero` method when the user clicks a hero's name.
-
-  **Two-way data binding** is an important fourth form
-  that combines property and event binding in a single notation, using the `ngModel` directive.
-  We didn't have a two-way binding in the `HeroListComponent` template;
-  here's an example from the `HeroDetailComponent` template:
-
-+makeExample('architecture/dart/lib/hero_detail_component.html', 'ng-model', 'lib/hero_detail_component.html (excerpt)')(format=".")
-:marked
-  In two-way binding, a data property value flows to the input box from the component as with property binding.
-  The user's changes also flow back to the component, resetting the property to the latest value,
-  as with event binding.
-
-  Angular processes *all* data bindings once per JavaScript event cycle,
-  depth-first from the root of the application component tree.
-  <!-- PENDING: clarify what "depth-first from the root" really means,
-  or reassure that they'll learn it soon. -->
-figure
-  img(src="/resources/images/devguide/architecture/component-databinding.png" alt="Data Binding" style="float:left; width:300px; margin-left:-40px;margin-right:10px" )
-:marked
-  We don't know all the details yet,
-  but it's clear from these examples that data binding plays an important role in communication
-  between a template and its component.
-<br clear="all">
-figure
-  img(src="/resources/images/devguide/architecture/parent-child-binding.png" alt="Parent/Child binding" style="float:left; width:300px; margin-left:-40px;margin-right:10px" )
-:marked
-  Data binding is also important for communication between parent and child components.
-<br clear="all">
-
-.l-main-section
-<a id="directive"></a>
-:marked
-  ## Directives
-figure
-  img(src="/resources/images/devguide/architecture/directive.png" alt="Parent child" style="float:left; width:150px; margin-left:-40px;margin-right:10px" )
-:marked
-  Angular templates are *dynamic*. When Angular renders them, it transforms the DOM
-  according to the instructions given by **directives**.
-
-  A directive is a class with directive metadata. In Dart we apply the `@Directive` annotation
-  to attach metadata to the class.
-<br clear="all">
-:marked
-  We already met one form of directive: the component. A component is a *directive-with-a-template*;
-  a `@Component` annotation is actually a `@Directive` annotation extended with template-oriented features.
-
-.l-sub-section
+block angular-library-modules
   :marked
-    While **a component is technically a directive**,
-    components are so distinctive and central to Angular applications that we chose
-    to separate components from directives in this architectural overview.
-:marked
-  Two *other* kinds of directives exist: _structural_ and _attribute_ directives.
+    Some modules are libraries.
```

Comment:
I was trying to mimic the TS prose. I just dropped that sentence since it isn't very helpful.